### PR TITLE
Revamped Installation Section (Part 2 - Docker/K8s)

### PR DESCRIPTION
**Pull request by domaincomponents**
Derived from Bert's blog and given structure as per the installation manual.

- Docker/K8s steps for Axon Server SE
- Docker/K8s steps for Axon Server EE

### CODE REVIEW

File: axon-server/installation/docker-k8s/axon-server-se.md [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+logging.file=/data/axonserver.log
+logging.file.max-history=10
+logging.file.max-size=10MB
+```
+
+### Customization
+
+The directory locations for the volumes can be specified as per your requirements. The image also has a third directory "/config" which is not marked as a volume. This gives you the capability to have an "axonserver.properties" file which can be placed in this location to override the above mentioned settings as well as add new properties similar to a local install.
+
+Assuming that you have a directory "_axonserverse**"**_ which will be the designated location for your volumes and configuration information.
```
Comment (bert-laverman) [inline axon-server/installation/docker-k8s/axon-server-se.md:64]: What is the purpose of the double-star notation?

File: axon-server/installation/docker-k8s/axon-server-se.md [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+
+Now if you query the API \(utilizing the “curl” command depicted above\) it will show that it is running with name “axonserver” and hostname “localhost”. Also the data directory will contain the ControlDB file, PID file, and a copy of the log output. The “events” directory will have the event and snapshot data.
+
+This completes a basic setup of the Axon Server SE Docker image with implementation of customizations.
+
+## Docker Compose
+
+Running Axon Server SE in docker-compose helps address more complex requirements around distributed scenarios. The following file will help start Axon Server SE with “./data”, “./events”, and “./config” mounted as volumes and the config directory is actually Read-Only.
+
+This again assumes that you have a directory "_axonserverse**"**_ which will be the designated location for your volumes and configuration information.
```
Comment (smcvb) [inline axon-server/installation/docker-k8s/axon-server-se.md:90]: The same "What is the purpose of the double-star notation?" from @bert-laverman would apply here.

File: axon-server/installation/docker-k8s/axon-server-se.md [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+    targetPort: 8124
+  clusterIP: None
+  selector:
+    app: axonserver
+---
+```
+
+Important to note here is that this is a pretty basic descriptor in the sense that it does not have any settings for the amount of memory and/or cpu to reserve for Axon Server SE which you may want to do for long-running deployments.
+
+To deploy, you would need a Kubernetes cluster and access to the [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) utility to help control these clusters. For a development Kubernetes cluster, it is recommended to use [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) or [Red Hat CodeReady Containers](https://github.com/code-ready/crc) which installs a Red Hat OpenShift Kubernetes cluster on your laptop . For production it is recommended to use a managed service like AWS [EKS](https://aws.amazon.com/eks/) / Google's [GKE](https://cloud.google.com/kubernetes-engine) or Azure's [AKS](https://azure.microsoft.com/en-us/services/kubernetes-service).
```
Comment (smcvb) [inline axon-server/installation/docker-k8s/axon-server-se.md:259]:
```suggestion
To deploy, you would need a Kubernetes cluster and access to the [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) utility to help control these clusters. For a development Kubernetes cluster, it is recommended to use [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) or [Red Hat CodeReady Containers](https://github.com/code-ready/crc) which installs a Red Hat OpenShift Kubernetes cluster on your laptop. For production it is recommended to use a managed service like AWS [EKS](https://aws.amazon.com/eks/) / Google's [GKE](https://cloud.google.com/kubernetes-engine) or Azure's [AKS](https://azure.microsoft.com/en-us/services/kubernetes-service).
```

File: extensions/tracing.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ With this instrumentation, we can chain synchronous and asynchronous commands an
 </dependency>
 ```
 
-The first dependency is [Spring Boot starter for Axon Tracing extension](../getting-started/maven-dependencies.md#axon-tracing-spring-boot-starter), which is the quickest start in to an extension configuration.
+The first dependency is [Spring Boot starter for Axon Tracing extension](https://github.com/domaincomponents/reference-guide/tree/375d9d62c18f30922cb3bc5be76351deeb9a64f0/getting-started/maven-dependencies.md#axon-tracing-spring-boot-starter), which is the quickest start in to an extension configuration.
```
Comment (smcvb) [inline extensions/tracing.md:21]: Why is the GitHub URL stuff needed for linking here?

File: release-notes/rn-axon-framework/rn-af-minor-releases.md [hunk 5]
```diff
@@ -6,61 +6,61 @@ Any patch release made for an Axon project is tailored towards resolving bugs. T
...
-   This was resolved in pull request [#1450](https://github.com/AxonFramework/AxonFramework/pull/1450).
-   
- * A small connectivity adjustment which was performed in the `AxonServerConnectionManager` for bug release 4.3.4 has been reverted.
- 
-   Although it worked successfully for some scenarios, it did not correctly cover all possibilities.
-   
-   The commit can be found [here](https://github.com/AxonFramework/AxonFramework/commit/5b9348040f4f977db3b9a15c3ae55904710814b6) for reference.
-   
-   The full scenario will be covered through the adjusted connector which is underway for beta release in 4.4.
+* The `TrackingEventProcessor#mergeSegment(int)` method was invoked with the high segment number of the pair to merge,
```
Comment (smcvb) [inline release-notes/rn-axon-framework/rn-af-minor-releases.md:9]: Would be interested to know what tool you use for adjusting the indentation here and adding the backslashes to the issue numbers. Asking this as I recently added these release notes and would like to maintain a consistent styling throughout our reference guide.

File: axon-server/installation/docker-k8s/README.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# Docker / K8s
+
+Axon provides support for Axon Server SE/EE to be run as Docker images utilizing plain Docker images / Docker Compose or larger-scale infrastructural products such as Kubernetes, Cloud-Foundry or Red Hat OpenShift.
+
+| Version |  |
+| :--- | :--- |
+| [Axon Server SE](axon-server-se.md) | Setup for Axon Server SE using Docker/Docker Compose and Kubernetes |
+| [Axon Server EE](axon-server-ee.md) | Setup for Axon Server EE using Docker/Docker Compose and Kubernetes |
+
+Axon provides [a github repo](https://github.com/AxonIQ/running-axon-server/tree/master/2-docker/1-docker-se) to help get started with using Docker/K8s for Axon Server SE/EE
```
[the remark this answers is not in the document]
Comment (bert-laverman) [reply axon-server/installation/docker-k8s/README.md:10]: Also, I would let this point to the whole repo, so remove everything starting at "`/tree/master`"

File: axon-server/installation/docker-k8s/axon-server-se.md (hunk 1 above)
Comment (domaincomponents) [reply axon-server/installation/docker-k8s/axon-server-se.md:64]: Italics

File: axon-server/installation/docker-k8s/axon-server-se.md (hunk 2 above)
Comment (domaincomponents) [reply axon-server/installation/docker-k8s/axon-server-se.md:90]: Italics

File: extensions/tracing.md (hunk 4 above)
Comment (domaincomponents) [reply extensions/tracing.md:21]: Resolved

File: release-notes/rn-axon-framework/rn-af-minor-releases.md (hunk 5 above)
Comment (domaincomponents) [reply release-notes/rn-axon-framework/rn-af-minor-releases.md:9]: Regular gitbook utility

**Pull request merged**